chain/src/tests/basic.rs

```diff
@@ -57,14 +59,12 @@ fn test_genesis_transaction_spend() {
         .witness(Script::default().into_witness())
         .input(CellInput::new(OutPoint::null(), 0))
         .outputs(vec![
-            CellOutput::new(
-                capacity_bytes!(100_000_000),
-                Bytes::default(),
-                Script::default(),
-                None
-            );
+            CellOutputBuilder::default()
+                .capacity(capacity_bytes!(100_000_000))
+                .build();
             100
         ])
+        .outputs_data(vec![Bytes::new(); 100])
```

Comment:
`output_data(Bytes::new())`

Reply:
`outputs_data.len()` must equal to `outputs.len()`, so here we need to set 100 length empty Bytes as outputs_data.